spec/spec_helper_acceptance.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require 'voxpupuli/acceptance/spec_helper_acceptance'
+require 'spec_helper_local'
```

Comment:
Why would you introduce a separate file for this? The file isn't managed now so you might as well include it here.